recipes/quickseries/LICENSE

```diff
@@ -0,0 +1,28 @@
+BSD 3-Clause License
```

Comment:
This file is in the tarball, no need to add it here.

Reply:
Deleted.